### ISSUE TITLE
Error on Fava for /api/errors

### ISSUE THREAD

**Issue by vkbytes** [beancount/fava#1568]
Lately I have been seeing this issue on the errors page and it doesn't open up any more
```
Starting Fava on http://127.0.0.1:5656
Exception on /beancount/api/errors [GET]
Traceback (most recent call last):
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/fava/json_api.py", line 153, in _wrapper
    return json_success(res)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/fava/json_api.py", line 59, in json_success
    return jsonify(
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/flask/json/__init__.py", line 342, in jsonify
    return current_app.json.response(*args, **kwargs)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/flask/json/provider.py", line 106, in response
    return self._app.response_class(self.dumps(obj), mimetype="application/json")
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/fava/core/charts.py", line 99, in dumps
    return ENCODER.encode(obj)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/simplejson/encoder.py", line 297, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/simplejson/encoder.py", line 379, in iterencode
    return _iterencode(o, 0)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/fava/core/charts.py", line 82, in default
    return JSONEncoder.default(self, o)
  File "/Users/vkbytes/miniforge3/lib/python3.9/site-packages/simplejson/encoder.py", line 273, in default
    raise TypeError('Object of type %s is not JSON serializable' %
TypeError: Object of type type is not JSON serializable
```

**Comment by vkbytes** [comment]
looks like the route to the endpoint is rendering 500

```
TypeError: Object of type type is not JSON serializable
192.168.64.1 - - [21/Mar/2023 00:51:26] "GET /beancount/api/errors HTTP/1.1" 500 -
Traceback (most recent call last):
```

**Comment by yagebu** [comment]
Should, like #1570, be fixed by #1574
